sections/remotes.tex

```diff
@@ -101,35 +77,17 @@ \subsection{Workflows / Branching Models}
 
 \end{frame}
 
-\subsection{Forking Workflow}
-\begin{frame}[fragile]
-  \subslidetitle
-
-  Now let's get our hands dirty in Open Source! \\
-  \vspace{1em}
-
-  Our goal is to:
-  \begin{itemize}
-    \item Use the \textit{Forking Workflow}
-    \item Contribute a change to a repository on \href{https://github.com}{GitHub}
-  \end{itemize}
-
-  \centerline{\includegraphics[width=\textwidth]{../assets/images/agile-ipsum.jpeg}}
-
-\end{frame}
-
-
 \subsection{Forking Workflow - Fork}
 \begin{frame}[fragile]
   \subslidetitle
 
-  Let's create a Fork from our friends awesome \textbf{agile-ipsum} project using \href{https://github.com}{GitHub}:
+  Let's create a Fork from the \textbf{gitmoon-colab} project using \href{https://github.com}{GitHub}:
```

Comment:
```suggestion
  Let's create a Fork from the \textbf{gitmoon-collab} project using \href{https://github.com}{GitHub}:
```

I'd rename the repository to `collab`, because I assume you wanted to abbreviate "collaboration" ...

Reply:
lets ask @randombenj :D but since we both can type i guess that was the intent